app/src/main/java/com/selfdrivingboat/boatcontroller/MainActivity.java

```diff
@@ -110,10 +110,14 @@ protected void onCreate(Bundle savedInstanceState) {
         initData();
         initService();
         logger.w("main activity started");
+
+        // TEMP CODE TO CALL CAMERA
+        SelfDrivingBoatCamera camera = new SelfDrivingBoatCamera(MyApplication.context());
+        camera.takePicture();
```

Comment:
if the code works here (when the app start) we will take a picture.
later we will move this two lines somewhere else. now leave them here for testing